packages/foreman/rubygem-dynflow/rubygem-dynflow.spec

```diff
@@ -1,101 +1,60 @@
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.3.0
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(multi_json)
-Requires: %{?scl_prefix}rubygem(msgpack) >= 1.3
-Requires: %{?scl_prefix}rubygem(msgpack) < 2
-Requires: %{?scl_prefix}rubygem(msgpack) >= 1.3.3
-Requires: %{?scl_prefix}rubygem(apipie-params)
-Requires: %{?scl_prefix}rubygem(algebrick) >= 0.7.0
-Requires: %{?scl_prefix}rubygem(algebrick) < 0.8
-Requires: %{?scl_prefix}rubygem(concurrent-ruby) >= 1.1.3
-Requires: %{?scl_prefix}rubygem(concurrent-ruby) < 1.2
-Requires: %{?scl_prefix}rubygem(concurrent-ruby-edge) >= 0.6.0
-Requires: %{?scl_prefix}rubygem(concurrent-ruby-edge) < 0.7
-Requires: %{?scl_prefix}rubygem(sequel) >= 4.0.0
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 2.3.0
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.3.0
+BuildRequires: ruby >= 2.3.0
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
-# Optional dependency that we always pull in
-Requires: %{?scl_prefix}rubygem(statsd-instrument)
-Requires: dynflow-utils
-
 %description
 Ruby workflow/orchestration engine.
 
 
 %package doc
-Summary: Documentation for %{pkg_name}
-Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}.
+Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
-
-%{?scl:scl enable %{scl} - << \EOF}
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
-%{?scl:EOF}
+%setup -q -n  %{gem_name}-%{version}
 
 %build
 # Create the gem as gem install only works on a gem file
-%{?scl:scl enable %{scl} - << \EOF}
-gem build %{gem_name}.gemspec
-%{?scl:EOF}
+gem build ../%{gem_name}-%{version}.gemspec
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
-%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
-%{?scl:EOF}
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -pa .%{gem_dir}/* \
+cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 %files
 %dir %{gem_instdir}
+%exclude %{gem_instdir}/.github
 %exclude %{gem_instdir}/.gitignore
-%exclude %{gem_instdir}/.github/install_dependencies.sh
-%exclude %{gem_instdir}/.github/workflows/ruby.yml
-%exclude %{gem_instdir}/.github/workflows/release.yml
 %exclude %{gem_instdir}/.rubocop.yml
 %exclude %{gem_instdir}/.rubocop_todo.yml
-%exclude %{gem_instdir}/Dockerfile
-%exclude %{gem_instdir}/docker-compose.yml
+%{gem_instdir}/Dockerfile
 %license %{gem_instdir}/MIT-LICENSE
 %{gem_instdir}/extras
 %{gem_libdir}
-%{gem_instdir}/web
+%exclude %{gem_instdir}/web
```

Comment:
I have some vague recollection that this is actually needed for the web console to function. Are you sure this can be excluded?